backend/src/app.ts

```diff
@@ -108,3 +127,33 @@ app.put("/converter/save", updateConverter);
 
 // compiler
 app.post("/compiler/compile", compileCode);
+
+// passport
+app.get("/auth/google", passport.authenticate("google", { scope: ["email", "profile"] }));
```

Comment:
Ja te handlery dla endpointów zrobiłem w app.ts, bo to była demonstracyjna wersja, przydałoby się te handlery przenieść do /user/enpoints.ts tak jak te pozostałe endpointy.